.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "Vendor/sqlcipher"]
```

Comment:
I've experienced the same thing, regardless of whether or not I use your way in this PR or use sqlcipher as a submodule.  Still working on it.  Thanks for the response.